challenges/10exercises.js

```diff
@@ -1,26 +1,122 @@
 // Define a function max() that takes two numbers as arguments and returns the largest of them.
 // Use the if-then-else construct available in Javascript.
+const max = (a,b) => {
+  if (a > b) {
+    return a
+  }
+  return b
+}
 
 // Define a function maxOfThree() that takes three numbers as arguments and returns the largest of them.
+const maxOfThree = (a, b, c) => {
+  if (a > b && a > c) {
+    return a
+  } else if (b > a && b > c) {
+    return b
+  } else {
+    return c
+  }
+}
 
 // Write a function that takes a character (i.e. a string of length 1) and returns true if it is a vowel, false otherwise.
+const vowelChecker = (letter) => {
+  return /[aeiou]/.test(letter)
+}
 
 // Write a function translate() that will translate a text into "rövarspråket".
 // That is, double every consonant and place an occurrence of "o" in between. For example, translate("this is fun") should return the string "tothohisos isos fofunon".
+const translate = (originalWord) => {
+  let origWordArray = originalWord.split('')
+  for (var i = 0; i < origWordArray.length; i++) {
+    if ( /[aeiou ]/.test(origWordArray[i]) ) {
+      i++
```

Comment:
You shouldn't have to increase i here in a for loop.  You should just do nothing for this iteration.